roles/example-cnf-app/templates/trex-cr.yaml.j2

```diff
@@ -14,5 +14,6 @@ spec:
   version: {{ app_version }}
   imagePullPolicy: {{ image_pull_policy }}
   networks: {{ networks_trex }}
```

Comment:
How are you managing the networks? It should be "cnf_app_networks" as testpmd is created with it.

Reply:
This is `packet_generator_networks` with additional trex mac address. https://github.com/rh-nfv-int/nfv-example-cnf-deploy/blob/v0.2/roles/example-cnf-app/tasks/trex-app.yaml#L22

Reply:
Yes, in that case, the same network should also be provided in the cnf_app_networks, are you adding it? I was thinking in case of deployment without LB, we can have only one parameter which can be used on both the places.

Reply:
No, but I tested 2 scenarios(same network and different network) and both worked. I thought we could still keep two network variables and let user provide same or different inputs. Am i missing anything ?

Reply:
It might have worked if it is all on the same interface and vlan. If we allow two different network inputs and user configures it with differnt vlans, then trex will be in one vlan and testpmd will be in another vlan, which will not work. So it is better to use only one definition on both ends.

Reply:
yes, you are right. lets pick `cnf_app_networks` if LB is not enabled.